### PR TITLE
fix: Reposition sign in canceled Snackbar

### LINKED ISSUE #869: Snackbar appearing on top of system navigation bar
**Describe the bug**

While going back from the sign in fragment, a snackbar pops up displaying "Sign in canceled!" This snackbar is overlapping over the system navigation bar.
**To Reproduce**
1. Go to Login fragment by selecting the profile icon from the Bottom Navigation VIew
2. Press the back button

**Screenshots**
ezgif com-video-to-gif

**Expected behavior**

The snackbar should be on top of the bottom navigation view

**Smartphone Info:**

|               | |
|---------------|-|
|Device         |HMD Nokia 6.1 Plus|
|Android Version|Pie 9.0 |

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by addiegupta**
Fixes #869 
Changes: 
The snackbar is now visible above the bottom navigation view

Screenshots for the change:

pr 870

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -20,6 +20,11 @@
             app:navGraph="@navigation/navigation_graph"
             app:layout_behavior="@string/appbar_scrolling_view_behavior" />
 
+        <androidx.coordinatorlayout.widget.CoordinatorLayout
+            android:id="@+id/mainSnackbar"
```
[the remark this answers is not in the document]
Comment (addiegupta) [reply app/src/main/res/layout/activity_main.xml]: That coordinatorLayout is being used to show the snackbar in the main activity. What should i rename it to?

File: app/src/main/java/org/fossasia/openevent/general/MainActivity.kt [hunk 2]
```diff
@@ -7,8 +7,7 @@ import androidx.navigation.NavController
 import androidx.navigation.fragment.NavHostFragment
 import androidx.navigation.ui.NavigationUI.setupWithNavController
 import com.google.android.material.snackbar.Snackbar
-import kotlinx.android.synthetic.main.activity_main.navigation
-import kotlinx.android.synthetic.main.activity_main.navigationAuth
+import kotlinx.android.synthetic.main.activity_main.*
```
Comment (GOVINDDIXIT) [inline app/src/main/java/org/fossasia/openevent/general/MainActivity.kt]: avoid wildcard imports

**Comment by addiegupta** [comment]
I have pushed another commit please review

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -20,6 +20,11 @@
             app:navGraph="@navigation/navigation_graph"
             app:layout_behavior="@string/appbar_scrolling_view_behavior" />
 
+        <androidx.coordinatorlayout.widget.CoordinatorLayout
+            android:id="@+id/mainSnackbarCoordinatorLayout"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"/>
```
Comment (iamareebjamal) [inline app/src/main/res/layout/activity_main.xml]: Not good, make it the wrapping element

Comment (addiegupta) [reply app/src/main/res/layout/activity_main.xml]: Okay thanks!

**Pull request merged**